documentation/bolt_configuration_options.md

```diff
@@ -25,6 +25,7 @@ ssh:
 -   `concurrency`: The number of threads to use when executing on remote targets. Default is `100`.
 -   `format`: The format to use when printing results. Options are `human` and `json`. Default is `human`.
 -   `hiera-config`: Specify the path to your Hiera config. The default path is `hiera.yaml` inside the [Bolt project directory](bolt_project_directories.md#).
+-   `trusted-external-command`: Specify the path to an executable that can produce [external trusted facts](https://puppet.com/docs/puppet/latest/release_notes_puppet.html#puppet-new-features-x.10.1).
```

Comment:
After talking to the coremmunity team I think we should document this as experimental for now.

Reply:
Updated here, in the CHANGELOG, and commit message.

Reply:
Updated here, in the CHANGELOG, and in the commit message.